kubernetes/helm_charts/core/notification/templates/deployment.yaml

```diff
@@ -41,6 +41,16 @@ spec:
         readinessProbe:
 {{ toYaml .Values.readinessProbe | indent 10 }}
         {{- end }}
+{{- $keys := .Files.Glob "keys/*" }}
```

Comment:
This block has indentation issues. Please refer this and update - https://github.com/project-sunbird/sunbird-devops/blob/63c1df34009e831d629ec1f7303c79bc09b5b432/kubernetes/helm_charts/core/groups/templates/deployment.yaml#L48-L56